pipenv/cli.py

```diff
@@ -721,6 +720,9 @@ def cli(ctx, where=False, venv=False, rm=False, bare=False, three=False, python=
 
         # --rm was passed...
         elif rm:
+            if not project.pipfile_exists:
```

Comment:
So this issue actually affects the `--venv` option as well. It would be best to get both of these cases handled at once.

I'd suggest we go something more along the lines of:

```python
if where:
    do_where(bare=bare)
    sys.exit(0)

if not project.pipfile_exists:
    click.echo(crayons.red('No Pipfile found. Type 'pipenv' for help creating a new project.'))
    sys.exit(1)
elif venv:
    ...
```